src/helpers/web3.ts

```diff
@@ -21,6 +22,7 @@ export async function getERC20TokenSymbol(networkId: number, tokenAddress: strin
   const provider = await handler.getFastestRpcProvider();
 
   // fetch token symbol
+  // @ts-expect-error expects a contract when imported through ESNext
   const contract = new ethers.Contract(tokenAddress, abi, provider);
```

Comment:
This doesn't seem right maybe @zugdev or @rndquu might know what to do

Reply:
Here are the errors:
![image](https://github.com/user-attachments/assets/35ad9cb5-2f9a-447a-a708-cdef0698ac19)
![image](https://github.com/user-attachments/assets/98756c0e-cb8d-45c5-a71b-a621d7d9bfb7)

Not sure what to do with these.